<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHotkeyDialog</class>
 <widget class="QDialog" name="CHotkeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit hotkey</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="gb_machine">
     <property name="title">
      <string>Machine</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QComboBox" name="cb_identifier"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="gb_hotkey">
     <property name="title">
      <string>Combination</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="3">
       <widget class="QPushButton" name="pb_advancedMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rb_press">
        <property name="text">
         <string>Press</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="pb_selectedHotkey">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>[Select]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rb_release">
        <property name="text">
         <string>Release</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QFrame" name="advancedFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="gb_action">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTreeView" name="tv_actions">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pb_accept">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pb_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
